openwisp_controller/config/static/config/js/vpn.js

```diff
@@ -0,0 +1,39 @@
+django.jQuery(function ($) {
+    'use strict';
+    if (!$('.add-form').length) {
+        return;
+    }
+
+    var showOverlay = function () {
+        var loading = $('#loading-overlay');
+        if (!loading.length) {
+            $('body').append(
+                '<div id="loading-overlay" class="djnjc-overlay loading"><div class="spinner"></div></div>'
+            );
+            loading = $('#loading-overlay');
+        }
+        loading.fadeIn(250, function () {
+            loading.css('display', 'flex');
+        });
+        setTimeout(function () {
+            var spinner = loading.find('.spinner');
+            spinner.fadeOut(250, function () {
+                var message = gettext(
+                    'Please be patient, we are creating all the necessary ' +
+                        'cyrptographic keys which may take a minute or two ...'
```

Comment:
Also, vscode wants to change indentation of this line. :-P

Reply:
jslinter is complaining, maybe we can try switching to jshint